src/main/java/com/flagsmith/FlagsmithConfig.java

```diff
@@ -114,6 +119,17 @@ public Builder sslSocketFactory(SSLSocketFactory sslSocketFactory, X509TrustMana
             return this;
         }
 
+        /**
+         * Add a custom HTTP interceptor.
+         *
+         * @param interceptor the HTTP interceptor
+         * @return the Builder
+         */
+        public Builder addHttpInterceptor(Interceptor interceptor) {
+            this.interceptor = interceptor;
```

Comment:
I've not used interceptors myself but from reviewing them, it looks as though it might be useful to allow multiple interceptors to be added? 

So we could keep the `addHttpInterceptor` builder method but just change the private variable to an array? 

Feel free to shoot me down here - this is slightly out of my comfort zone :)

Reply:
Hi @matthewelwell, yes this sounds like a good idea, I'll make it a list.